app/src/main/java/com/example/pokemon/presenters/MainPresenter.kt

```diff
@@ -79,7 +89,8 @@ class MainPresenter
         response.let {
             if (it.isNotEmpty()) {
                 pokeList = ArrayList(it)
-                mView?.resetPokemonList(it)
+                mView?.setListToAdapter(it)
+                mView?.showPokemonRV()
```

Comment:
why do we need to showPokemonRV ? whenever we are adding to adapter we are already showing right ?